pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -480,6 +492,7 @@ func (asw *actualStateOfWorld) DeleteVolumeNode(
 
 	volumeObj, volumeExists := asw.attachedVolumes[volumeName]
 	if !volumeExists {
+		// TODO: Always call removeVolumeFromReportAsAttached?
```

Comment:
if volume not exist in attachedVolumes, that means DeleteVolumeNode must be called, which already calls removeVolumeFromReportAsAttached